mxcube3/routes/Login.py

```diff
@@ -80,15 +94,11 @@ def loginInfo():
             return make_response("", 409)
 
         # auto log in
-        loginInfo["loginRes"] = limsutils.lims_login(loginID, loginInfo["password"])
+        #loginInfo["loginRes"] = limsutils.lims_login(loginID, loginInfo["password"])
```

Comment:
Because it was as I understood it not needed, the only thing lims_login does is to login to LIMS. We don't really need to do that if the user is already logged in. We, should, only need to login to LIMS when the users logs in. The auto login feature should just keep the session information for the "current" user. It seems unnecessary to login to LIMS twice, but maybe I'm mistaken.